docs/concepts-guidelines.md

```diff
@@ -127,7 +127,7 @@ jobs:
   example:
     runs-on: ubuntu-latest
     # Check if the event is not triggered by a fork
-    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Good to know about `.fork` 👍 

The brackets, `${{` and `}}` are unnecessary, though. I would prefer to leave those out.